src/Fixer/Operator/TernaryToElvisOperatorFixer.php

```diff
@@ -0,0 +1,265 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Operator;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class TernaryToElvisOperatorFixer extends AbstractFixer
+{
+    /**
+     * Lower precedence and other valid preceding tokens.
+     *
+     * @var array
+     */
+    private static $validBeforeEndTypes = [
```

Comment:
This can be a constant, right? Seeing variable always suggests me that it can be updated in the runtime and this one is not.

Reply:
thanks, done!